app/api/__init__.py

```diff
@@ -173,3 +178,9 @@
 api.route(DiscountCodeDetail, 'discount_code_detail', '/discount-codes/<int:id>')
 api.route(DiscountCodeRelationship, 'discount_code_event',
           '/discount-codes/<int:id>/relationships/event')
+
+# speakers
+api.route(SpeakerList, 'speaker_list', '/events/<int:event_id>/speakers')
```

Comment:
Speakers should also be accessible at `/events/<identifier>/speakers` 
(ie) both event ID or identifier could be used.